### PR TITLE
[amharic] Update links for Gurage/Sebatbeit Blin and Geez keyboards

### PR DESCRIPTION
**Pull request by LornaSIL**
These changes were made at the request of @dyacob.

### CODE REVIEW

File: amharic/index.php [hunk 1]
```diff
@@ -129,10 +129,11 @@
   <ul>
       <li><a href="/keyboards/gff-awn-powerpack-7">Awngi</a></li>
       <li><a href="/keyboards/gff-bcq-powerpack-7">Bench</a></li>
-      <li><a href="/keyboards/gff-byn-powerpack-7">Blin</a></li>
-      <li><a href="/keyboards/gff-gez-powerpack-7">Ge'ez</a></li>
+      <li><a href="/keyboards/gff_blin">Blin</a></li>
+      <li><a href="/keyboards/https://keyman.com/keyboards/gff_geez">Ge'ez</a></li>
```
Comment (mcdurdin) [inline amharic/index.php:133]:
```suggestion
      <li><a href="/keyboards/gff_geez">Ge'ez</a></li>
```

File: tigrigna/eritrean.php [hunk 2]
```diff
@@ -115,10 +115,11 @@
         <li><a href="/tigrigna/">Tigrigna (Ethiopia)</a></li>
         <li><a href="/keyboards/gff-awn-powerpack-7">Awngi</a></li>
         <li><a href="/keyboards/gff-bcq-powerpack-7">Bench</a></li>
-        <li><a href="/keyboards/gff-byn-powerpack-7">Blin</a></li>
-        <li><a href="/keyboards/gff-gez-powerpack-7">Ge'ez</a></li>
+        <li><a href="/keyboards/gff_blin">Blin</a></li>
+        <li><a href="/keyboards/https://keyman.com/keyboards/gff_geez">Ge'ez</a></li>
```
Comment (mcdurdin) [inline tigrigna/eritrean.php:119]:
```suggestion
        <li><a href="/keyboards/gff_geez">Ge'ez</a></li>
```

File: tigrigna/index.php [hunk 3]
```diff
@@ -115,10 +115,11 @@
         <li><a href="eritrean.php">Tigrigna (Eritrea)</a></li>
         <li><a href="/keyboards/gff-awn-powerpack-7">Awngi</a></li>
         <li><a href="/keyboards/gff-bcq-powerpack-7">Bench</a></li>
-        <li><a href="/keyboards/gff-byn-powerpack-7">Blin</a></li>
-        <li><a href="/keyboards/gff-gez-powerpack-7">Ge'ez</a></li>
+        <li><a href="/keyboards/gff_blin">Blin</a></li>
+        <li><a href="/keyboards/https://keyman.com/keyboards/gff_geez">Ge'ez</a></li>
```
Comment (mcdurdin) [inline tigrigna/index.php:119]:
```suggestion
        <li><a href="/keyboards/gff_geez">Ge'ez</a></li>
```

**Review by mcdurdin — CHANGES_REQUESTED** [review]
A few malformed links to fix but apart from that looks good.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**